Sources/CCFilter/CCFilter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public class CCFilter {
+    private var root: CCFilterNode
+
+    public init(root: CCFilterNode) {
+        self.root = root
+    }
+}
+
+public extension CCFilter {
+    var urlEncoded: String {
+        return root.urlItems.joined(separator: "&")
```

Comment:
I guess we need to URL encode each of the values, not just join them by `&`.

I.e. passing `Søk & test` as value to a node should return `q=S%C3%B8k%20%26%20test`, but now returns `q=Søk & test`. This will break the search😅

Reply:
Can we perhaps use `URLQueryItem` instead, it will do url encoding for us?

Reply:
I fix!

Reply:
Maybe we could return an array of `URLQueryItem` in `urlItems` instead of strings?

Reply:
Yes, much better to use `URLQueryItem` 👍